### PR TITLE
Restructure Monitors and Notifiers sections

### PR DESCRIPTION
**Pull request by manototh**
Dependency: [PR changing the links in the app](https://github.com/axiomhq/app/pull/3210)

Previews:

Monitors

- [Overview](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/monitors)
- [Configure](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/configure-monitors)
- [Anomaly](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/anomaly-monitors)
- [Match](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/match-monitors)
- [Threshold](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/threshold-monitors)
- [Examples](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/monitor-examples)

Notifiers

- [Overview](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/notifiers-overview)
- [Configure](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/configure-notifiers)
- [Custom Webhook](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/custom-webhook-notifier)
- [Discord](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/discord-notifier)
- [Email](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/email-notifier)
- [Microsoft Teams](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/microsoft-teams-notifier)
- [Opsgenie](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/opsgenie-notifier)
- [PagerDuty](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/pagerduty)
- [Slack](https://axiom-mano-monitors-restructure.mintlify.app/monitor-data/slack-notifier)

### CODE REVIEW

File: monitor-data/monitor-examples.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Notify on all occurrences of error
+
+To receive a notification on all occurrences of an error, create a match monitor where the filter conditions match the events reporting the error.
+
+To receive only certain attributes in the notification message, use the `project` operator.
+
+## Notify when error rate above threshold
+
+To receive a notification when the error rate exceeds a threshold, create a threshold monitor with an APL query that identifies the rate of error messages.
```
Comment (dominicchapman) [inline monitor-data/monitor-examples.mdx]: Would it help build a relationship between Examples and Monitor types > Type pages if we made create a [threshold monitor](url) a link?

File: monitor-data/monitor-examples.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Other options:
+- To trigger the monitor when the count is above or equal to 10 for any individual message type, set the threshold to 10 and the comparison operator to **above or equal**.
+- To run the monitor every 5 minutes, set the frequency to 5.
+- To run the monitor the query with a range of 10 minutes, set the range to 10.
+
+By default, the monitor enters the alert state when any of the counts returned by the query cross the threshold, and remains in the alert state until no counts cross the threshold. To alert separately for each message value instead, enable **Notify by group**.
+
+## Notify when response times spike
+
+To receive a notification whenever your response times spike without having to rely on a single threshold, create an anomaly monitor with an APL query that tracks your median response time.
```
Comment (dominicchapman) [inline monitor-data/monitor-examples.mdx]: As above for create an [anomaly monitor](url).

**Review by dominicchapman — APPROVED** [review]
This feels much more approachable to me than a looooong all encompassing page. Nice job 👍

File: monitor-data/monitor-examples.mdx (hunk 1 above)
Comment (manototh) [reply monitor-data/monitor-examples.mdx]: Yes, really good point, thanks!

**Pull request merged**